oop/lesson1/date.cpp

```diff
@@ -35,6 +35,22 @@ void Date::setDay(int day) {
     mDay = day;
 }
 
+int Date::getMonth() const {
+    return mMonth;
+}
+
+void Date::setMonth(int month){
```

Comment:
space between ) and {
```suggestion
void Date::setMonth(int month) {
```